src/open_inwoner/templates/pages/category/detail.html

```diff
@@ -4,7 +4,7 @@
 {% block header_image %}
 {% if object.image %}
     <aside class="banner" aria-hidden="true">
-        <img class="main-image main-image--start-after" src="{{ object.image.url }}" alt="{{ object.image.default_alt_text|default:'Header afbeelding' }}">
+        <img class="main-image main-image--start-after" src="{{ object.image.url }}" alt="{{ object.image.default_alt_text|default:'' }}">
     </aside>
```

Comment:
See previous comment/question.

Reply:
@pi-sigma this is a bit different: this one is either empty by default, or will actually have content if the admin sets this up in the 'mappen' configuration for images: http://127.0.0.1:8000/admin/filer/folder/. I am not sure if those config fields are translatable.
The ALT attribute must stay empty **if** they are decorative images, unless the admin/webeditor feels the image is not decorative and actually conveys important information.